### ISSUE TITLE
Microsoft.ServiceFabric.Client.Http v4.9.0: Could not load file or assembly 'Newtonsoft.Json, Version=13.0.0.0

### ISSUE THREAD

**Issue by johncrim** [Microsoft/service-fabric-client-dotnet#122]
**Describe the bug**
Microsoft.ServiceFabric.Client.Http v4.9.0 is compiled against Newtonsoft.Json v13.0.0.0; but the nuget package refererences Newtonsoft.Json v10.0.0.0. So any builds/packages will include 10.0, but will result in a runtime failure like:

```
Unhandled exception: System.IO.FileNotFoundException: Could not load file or assembly 'Newtonsoft.Json, Version=13.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed'. The system cannot find the file specified.

File name: 'Newtonsoft.Json, Version=13.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed'
   at Microsoft.ServiceFabric.Client.Http.ClusterClient.GetClusterHealthAsync(Nullable`1 nodesHealthStateFilter, Nullable`1 applicationsHealthStateFilter, Nullable`1 eventsHealthStateFilter, Nullable`1 excludeHealthStatistics, Nullable`1 includeSystemApplicationHealthStatistics, Nullable`1 serverTimeout, CancellationToken cancellationToken)
```

**To Reproduce**
Using asp.net core 7.0, build any sample executable that uses Microsoft.ServiceFabric.Client.Http v4.9.0 and calls a ServiceFabric cluster.  Package the executable and run it.

**Expected behavior**
Not this

**Additional context**
Here's the [nuspec from the nupkg](https://www.nuget.org/packages/Microsoft.ServiceFabric.Client.Http#dependencies-body-tab):

```xml
<?xml version="1.0" encoding="utf-8"?>
<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
  <metadata>
    <id>Microsoft.ServiceFabric.Client.Http</id>
    <version>4.9.0</version>
...
    <dependencies>
      <dependency id="Newtonsoft.Json" version="10.0.3" />
    </dependencies>
  </metadata>
</package>
```

Here's a screenshot from dotpeek:
![image](https://github.com/microsoft/service-fabric-client-dotnet/assets/3312127/c9af2277-5e4f-4b06-8daa-f414e21e8139)

**Comment by johncrim** [comment]
The workaround is to add a reference to Newtonsoft.Json 13.x for any projects using Microsoft.SErviceFabric.Client.Http.

**Comment by jeffj6123** [comment]
@johncrim thank you for reporting this. we will get a new version released to fix this

**Comment by jeffj6123** [comment]
@johncrim  we have released 4.10.1, which should fix the dependency issue.

**Comment by johncrim** [comment]
Verified that the issue is fixed in 4.10.1.
